### DISCUSSION TITLE
I cannot manipulate atcb-list-items styles

### DISCUSSION THREAD

**Discussion by davidpino-gj** [add2cal/add-to-calendar-button#103] [Q&A] [answered]
Hello, first of all, thank you for making this, I find it a very friendly and usable package to develop with.

Question, is it possible to customize or modify the list-items styles and other elements that render in the DOM after the `atcb_action()` is executed?.

I'm currently working with this in React and I'm having a hard time trying to make it look as requested.

Thank you.

**Comment by jekuer** [comment]
Planned in the way, that you can choose from different style options per button. This shall include a "none" option, where you would need to define the style completely on your own.
But need to see how it works out. Will be a little bit gambling with restrictions.

(also covered in #97 )

**Comment by jekuer** [comment, marked as answer]
There is now a documented way to change the style of more or less all things.
Check out https://add-to-calendar-button.com/en/advanced-use#1
